lang/en.lyaml

```diff
@@ -168,6 +169,7 @@ en:
       subcommands:
         migrate:
           describe: "Migrate from the deprecated {{ deprecatedConfigPath }} file to the new global config file at {{ globalConfigPath }}."
+          verboseDescribe: "This command will create a new global configuration file at the root of your working directory and rename the deprecated config to {{ archivedConfigPath }}. If any conflicting values are detected, we will prompt you to reconcile them unless the \"--force\" flag is supplied."
```

Comment:
I think this should be "in your root directory" - the location of the config file doesn't have anything to do with where your current working direcotry is